dist/fulcrum-schema.min.js

```diff
@@ -3120,6 +3126,12 @@ Schema.systemRepeatableTableColumns = [{
 }, {
   name: 'parent_resource_id',
   type: 'string'
+}, {
+  name: 'record_record_key',
+  type: 'string'
+}, {
+  name: 'record_record_sequence',
+  type: 'integer'
```

Comment:
@jahndis made a good point: maybe these should be "parent_record_xxx"?

The idea is that repeatables would have a copy of the parent/root records key and sequence. Not sure whether that's what the code is actually doing, but that's the intent.

Reply:
So, it turns out "record" == "root" while "parent" == "one level up". For example, "id" is the current records id, while "parent_record_id" is the immediate parents id and "record_id" is the root record id, which could be grand or great-grand parent. Based on that, "record_record_key" is probably the correct term.